packaging/scripts/postinstall.sh

```diff
@@ -6,6 +6,67 @@ ETC_SERVICE="/etc/systemd/system/evcc.service"
 USR_LOCAL_BIN="/usr/local/bin/evcc"
 RESTART_FLAG_FILE="/tmp/.restartEvccOnUpgrade"
 
+
+# Call /usr/bin/evcc checkconfig and capture the output
+# if exit code is 0, then remove /tmp/oldevcc
+# if exit code is not 0, then fail installation with error message and copy /tmp/oldevcc back to /etc/evcc
+# if /tmp/oldevcc does not exist, then do nothing
+failInstallation=0
+INTERACTIVE=0
+# is shell script interactive?
+if [ -t 0 ]; then
+  INTERACTIVE=1
+else
+  INTERACTIVE=0
+fi
+
+if [ -f /tmp/oldevcc ] && [ $INTERACTIVE -eq 1 ]; then
+    checkConfigOutput=$(/usr/bin/evcc checkconfig 2>&1 || true)
+	oldevccversion=$(/tmp/oldevcc -v)
+	newevccversion=$(/usr/bin/evcc -v)
+    if echo "$checkConfigOutput" | grep -q "config valid"; then
```

Comment:
if [ "$checkConfigOutput" = "config valid" ]

Reply:
Solche Änderungen kannst Du auch direkt inline zum applien vorschlagen ;)

Reply:
That makes no sense at this point. If successful, the checkconfig command does not return exactly "config valid", but also dynamic information such as the version number. It should therefore be checked for contains and not for equals.

Reply:
Sorry about that. I checked what the command does, but apparently I did not look close enough. My bad.